<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QListWidget" name="listWidget"/>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="toolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QFontComboBox" name="fontComboBox"/>
            </item>
            <item row="1" column="1">
             <widget class="QTableWidget" name="tableWidget">
              <row>
               <property name="text">
                <string>New Row</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>dfdfdfdfss</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>New Column</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>New dfddfdf3</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>reter</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>erter</string>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string>erter</string>
               </property>
              </item>
              <item row="1" column="1">
               <property name="text">
                <string>erte</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QProgressBar" name="progressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QSlider" name="horizontalSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>c</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
